app/views/guests/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h3>Name: <%=@guest.name%></h3>
+
+<%@rooms.each do |room|%>
+<div id="room_info-<%=room.id%>">
+  <p>Suite: <%=room.suite%></p>
+  <p>Rate: <%=room.rate%></p>
+  <p>Hotel: <%=room.hotel.name%></p>
+<%end%>
+
+<h5>Add Room:</h5>
+<%= form_tag("/guests/#{@guest.id}/guest_rooms", local: true)%>
+<%= text_field_tag(:room_id, params[:room_id])%>
+<%= submit_tag('Submit')%>
```

Comment:
Great job setting up your form. Again, route should point to `"/guests/#{@guest.id}/rooms"`, but everything else is great. I would also add a `label_tag` as well.